src/styles/blocks/style-block-card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+}
+
+.card--size {
+  box-sizing: border-box;
+  width: 200px;
+
+}
+
+.card__image {
+  display: flex;
+  background-size: cover;
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: hsl(234, 80%, 12%);
+  width: 166px;
+  /* height: 36px; */
+  margin: 0 16px 4px;
```

Comment:
Don't hardcode width for text content, just add padding 32px 16px 16px to whole card like in mockup